### ISSUE TITLE
Missing compose functions in material3

### ISSUE THREAD

**Issue by Thomas-Vos** [JetBrains/compose-multiplatform#2037]
The following functions seem to not have been added yet in material3 for desktop, but are available for Android. It would be great if they could be added so I do not have to copy the material3 source code into my project, and keep that up to date.

- `androidx.compose.material3.AlertDialog`
- `androidx.compose.material3.DropdownMenu`
- `androidx.compose.material3.DropdownMenuItem`
- `androidx.compose.material3.ExposedDropdownMenuBox`

**Comment by Thomas-Vos** [comment]
For exposed dropdown menus I found this issue: https://issuetracker.google.com/issues/201051696

**Comment by CXwudi** [comment]
Any progress on `AlertDialog`? 👀

**Comment by QiXi** [comment]
> 
To use AlertDialog on desktop, you can install two dependencies at the same time.
```
api(compose.material) //for AlertDialog 
api(compose.material3)
```
as a temporary solution

**Comment by fflopsi** [comment]
If I'm not mistaken, `androidx.compose.material3.DatePickerDialog` is also one of the composables that is still missing

**Comment by MatkovIvan** [comment]
@fflopsi It was tracked in #3354, #1299. We merged it recently https://github.com/JetBrains/compose-multiplatform-core/pull/717

**Comment by alexzhirkevich** [comment]
> If I'm not mistaken, androidx.compose.material3.DatePickerDialog is also one of the composables that is still missing

@MatkovIvan 
Actually, he has a point. I've added support for date picker itself, but [DatePickerDialog](https://github.com/JetBrains/compose-multiplatform-core/blob/c63c81e0108d8799cff51ae335344919b0d2c423/compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/DatePickerDialog.android.kt#L65) that uses AlertDialog under the hood, is still not merged to common

**Comment by MatkovIvan** [comment]
Right, it's about dialog, not a picker itself. Well, `material3.AlertDialog` is already [merged](https://github.com/JetBrains/compose-multiplatform-core/pull/710), so it looks like nothing prevents to "commonize"  it too. I'll make sure that this dialog is also in our queue.

**Comment by MatkovIvan** [comment]
All components mentioned in this issue were merged into main branch, closing

**Comment by helmysaeed** [comment]
in which release we can find them pleas

**Comment by MatkovIvan** [comment]
The next one, but most part of it is already available in 1.5.0

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2037">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
